axum/Cargo.toml

```diff
@@ -68,6 +68,7 @@ tower-service = "0.3"
 # optional dependencies
 axum-macros = { path = "../axum-macros", version = "0.5.0-rc.1", optional = true }
 base64 = { version = "0.22.1", optional = true }
+form_urlencoded = { version = "1.1.0", optional = true }
```

Comment:
I would prefer not to add this explicit extra dependency too, but `serde_urlencoded` currently has no way to construct a `Deserializer` without it and we need a `Deserializer` instance to pass into `serde_path_to_error`.

Reply:
Since we're switching to `serde_html_form` anyways, if you have an idea for how to make the `Deserializer` publically constructible in a nice way (haven't looked at that code in a while myself), feel free to make a PR.

Reply:
Great. Let's wait until after https://github.com/tokio-rs/axum/issues/3075 then, and then you can update the PR to use that (I released v0.2.7 of `serde_html_form` with this).

Reply:
With #3075 postponed, I think we can leave things as they are here. Upstreaming the constructor to `serde_urlencoded` wouldn't hurt, but isn't really necessary either.